CMakeLists.txt

```diff
@@ -12,7 +12,12 @@ option(USE_VCPKG "Use VCPKG to download and manage dependencies" OFF)
 
 if (USE_VCPKG)
     # Detect correct triplet based on CMake env
-    if (WIN32)
+    if (CMAKE_SYSTEM_NAME STREQUAL "Mingw")
+        set(VCPKG_TARGET_TRIPLET "x64-mingw-static")
+        set(VCPKG_INSTALL_OPTIONS "--allow-unsupported")
+        set(PTHREAD_LIBRARY "-lpthread")
+        set(WIN32 1)
```

Comment:
This is provided by cmake no?

Reply:
Yeah, it's provided by cmake.  I removed it from my pull request.